### ISSUE TITLE
Play 11 Needs Accessibility (Section 508 Compliance) added

### ISSUE THREAD

**Issue by normanbrobinson** [WhiteHouse/playbook#45]
The three policy oriented items that are critical to government facing applications includes Security, Privacy, and Accessibility (Section 508 compliance, specifically). That needs to be added as it is a critical gap missing from this playbook. 

It should not be assumed. Yes, it is part of standards. Yes, it is part of acquisition and selection. And yes, it is often a failure when people "choose commercial technologies" as they have not been addressing the mandate as the government has been required to do, and have not been holisticly impacted by ADA lawsuits for failiing to support the needs of people with disabilities, or the adaptation and modification of content by end users such as those using assistive techologies
